views/articles/new.ejs

```diff
@@ -0,0 +1,13 @@
+<% include ../partials/header.ejs %>
+
+  <h1>New Article</h1>
+
+  <form action="/articles" method="POST">
+    <input type="text" name="title" placeholder="title">
+    <input type="text" name="body" placeholder="body"> 
+    <input type="submit">
+  </form>
+
```

Comment:
here is an example of some code to perform a search.

app.get('/search', function(req, res) {
var searchTerm = req.query.search;
var articles = fs.readFileSync('./data.json');
articles = JSON.parse(articles);
var searchResults = searchArticles(articles, term);
res.render('articles/searchResults.ejs', { searchResults: searchResults });
});